lib/auth.ts

```diff
@@ -1,6 +1,7 @@
 import * as jwt from 'jsonwebtoken';
 import { NextApiRequest } from 'next';
 import * as BigCommerce from 'node-bigcommerce';
+import { testApiConfig } from '../test/utils';
```

Comment:
I feel like calling this "test" implies that it's only used for automated tests, when actually this is just a general api config, isn't it? (I have to be honest I actually thought we'd already basically added this). Any reason not to just call it apiConfig?